packages/paste-core/primitives/combobox/stories/index.stories.tsx

```diff
@@ -175,24 +171,17 @@ export const BasicMultiCombobox: React.FC = () => {
     selectedItems,
   } = useMultiSelectPrimitive({});
 
-  const handleSelectItemOnClick = React.useCallback(
-    ({selectedItem}) => {
-      addSelectedItem(selectedItem);
-
-      setFilteredItems((currentFilteredItems) => currentFilteredItems.filter((item) => item !== selectedItem));
-    },
-    [addSelectedItem, setFilteredItems]
-  );
-
   const handleRemoveItemOnClick = React.useCallback(
     (selectedItem) => {
       removeSelectedItem(selectedItem);
-
-      setFilteredItems((currentFilteredItems) => [...currentFilteredItems, selectedItem].sort());
     },
     [removeSelectedItem]
   );
 
+  const getFilteredItems = (): string[] => {
```

Comment:
Using a function to filter the list makes typeahead easier than actually adding/removing items in the state.